NOTES.md

```diff
@@ -0,0 +1,13 @@
+## 1. How do I set up my local environment to run the app locally?
+
+- Refer to README.md / check out package.json
+- you may need to updated your node version to run the project
+  ()
+
+## 2. What is Chakra UI, and where can I go to learn more about it?
+
+- https://chakra-ui.com/
+- readily accesible styling components that can be imported for React application
+
+## 3. How to maintain version control
+<img width="1462" alt="Screen Shot 2021-09-18 at 8 03 01 am" src="https://user-images.githubusercontent.com/83195319/133946101-5d7346c8-e1c0-49ef-9721-465cf7bc680c.png">
```

Comment:
I'm saying this only for the future, and this is being very pedantic, but would be good to have only the changes relating to this ticket in here. Super super pedantic. Please do not change this though - the README work is awesome and I'm really glad it's happening.

Reply:
Hi Matt,

Thanks for the feedback! Qing has explained it well during tonight's dev meeting - and I am aware of the confusion it made when my pull request are not relevant to my commits.

It will be much easier for the dev team to review my PR to keep the changes relevant to the branch! I know what not to do from now on :)